core/bench_test.go

```diff
@@ -157,22 +158,17 @@ func benchInsertChain(b *testing.B, disk bool, gen func(int, *BlockGen)) {
 
 	// Generate a chain of b.N blocks using the supplied block
 	// generator function.
-	gspec := Genesis{
+	gspec := &Genesis{
 		Config:   params.TestChainConfig,
 		Alloc:    GenesisAlloc{benchRootAddr: {Balance: benchRootFunds}},
-		Coinbase: constants.BlackholeAddr,
+		GasLimit: params.TestChainConfig.FeeConfig.GasLimit.Uint64(),
```

Comment:
why did we add gaslimit?

Reply:
TestChainConfig enables Subnet-EVM fork and Subnet-EVM Fork checks if header.GasLimit == config.GasLimit.